pkg/generate/app/pipeline.go

```diff
@@ -91,7 +91,7 @@ func (p *Pipeline) NeedsDeployment(env Environment, name string) error {
 // Objects converts all the components in the pipeline into runtime objects
 func (p *Pipeline) Objects(accept, objectAccept Acceptor) (Objects, error) {
 	objects := Objects{}
-	if p.InputImage != nil && p.InputImage.AsImageStream && accept.Accept(p.InputImage) {
+	if p.InputImage != nil && accept.Accept(p.InputImage) {
```

Comment:
@bparees when we setup the from field in the buildConfig pointing to an imageStream and then start-build, that imageStream gets validated and we need to have it in our registry right?